packages/dappmanager/src/daemons/avahi/index.ts

```diff
@@ -35,6 +41,12 @@ class AvahiController {
         return;
       }
 
+      const isAvahiInstalled = await isAvahiDaemonInstalled();
+      if (!isAvahiInstalled) await installAvahiDaemon();
+
+      const isAvahiRunning = await isAvahiDaemonRunning();
+      if (!isAvahiRunning) await startAvahiDaemonOnHost();
```

Comment:
We should be able to refactor this into doing a single shellHost call

Reply:
@pablomendezroyo The point of having a script was to not call shellHost 4 times but only 1

Reply:
I know, I just did the script, how and when to call it still need to be implemented

The idea would be:
- Call avahi_daemon.sh  --initialize on dappmanager startup, this would ensure:
    -  Install `avahi-tools` if necessary
    - Edit `/etc/hosts` and `/etc/avahi/avahi-daemon.conf` when hostname != dappnode. This situation might happen when install from script.
    - Start and enable avahi-daemon
- Use the other flags (`--start` `--stop` `--restart` `--status`) on demand by the user in the UI